OWML.Launcher/SocketUtils.cs

```diff
@@ -9,35 +9,42 @@
 
 namespace OWML.Launcher
 {
-    public class SocketListener
+    public class SocketUtils
```

Comment:
this kind of name kinda suggests a static class. i don't have any better ideas at the top of my head so this isn't a blocker ;-)

Reply:
Actually, maybe SocketListener is still a good choice?